lib/modules/datasource/docker/index.spec.ts

```diff
@@ -35,6 +39,9 @@ function mockEcrAuthReject(msg: string) {
 
 describe('modules/datasource/docker/index', () => {
   beforeEach(() => {
+    if (GoogleAuth) {
+      (GoogleAuth as any).mockReset();
```

Comment:
we've a mocked test util to mock with types, search code

Reply:
I'm not able to get it to work with `mocked`. The closest I came required me to mock all methods of GoogleAuth, which I don't think is desired.

Reply:
Updated the tests to make use of mocked, not sure if this is the desired solution.